Source/Core/VideoBackends/D3D/TextureCache.cpp

```diff
@@ -261,6 +265,27 @@ void TextureCache::CopyEFBToCacheEntry(TCacheEntry* entry, bool is_depth_copy,
   else
     D3D::SetPointCopySampler();
 
+  struct PixelConstants
+  {
+    float filter_coefficients[3];
+    float gamma_rcp;
+    float clamp_top;
+    float clamp_bottom;
+    float pixel_height;
+    u32 padding;
+  };
+  PixelConstants constants;
+  for (size_t i = 0; i < filter_coefficients.size(); i++)
+    constants.filter_coefficients[i] = filter_coefficients[i] / 64.0f;
```

Comment:
Maybe we should move this `64` to a variable we can reference somewhere with a name?  Something like `vertical_filter_max` and then copy phire's original comment about how this number is what games should never exceed but some due to affect the brightness?

Reply:
That is just called 0.6 fixed-point format. I wonder a bit about the the restriction to not allow values bigger or equal to 1..

Reply:
I've verified this on hardware; to not affect the intensity of the colour the three coefficients which we sum together must not exceed 64. Smaller than 64 results in a decrease in intensity, and greater than 64 increases the intensity as you would expect.

The previous and next rows can be set to 0..63 (2 six-bit values), the "current/middle" row to 0..95 (3 six-bit values). So at maximum a 1.5x increase in intensity for a single line, or 3.5x over the three lines. I'm not sure if the hardware multiplies each coefficient by the incoming colour or sums it first like I'm doing, and I've no idea if this is something we can even verify, as it's all clamped afterwards.

Reply:
There is no scaling by 64 in fixed point format, this is just an reinterpretation of the data. So we have to talk about the truncation instead. If you already implement the 6*8 bit multiplications, you may as well add the accumulation with the same precision. So I vote for the full precisition convolution and the 6 bit shift afterwards.

Reply:
Ah, so if I'm understanding this right, 0.6*8.0+0.6*8.0+0.6*0.8 will give the same results as (0.6+0.6+0.6) -> 0.8*0.8 regardless of the intermediate precision?